tests/e2e/tslint.json

```diff
@@ -37,10 +37,9 @@
     ],
     "member-ordering": [
       true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
-    ],
+      {
+        "order": "statics-first"
+      }         ],
```

Comment:
Code style notice: it would simpler to add new property into this json array if if keep `],` in the separate line (see lines 108-109 below as an example)

Reply:
fixed